quickstart.md

```diff
@@ -12,13 +12,72 @@ To use components from KCP-Edge you must:
 - [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
 - A Kubernetes cluster (for local testing, consider [kind](http://kind.sigs.k8s.io))
 
+#### Example (Centos)
+
+##### install git, install go version 1.20, set environment variables 
+```
+dnf install git
```

Comment:
should we also install 'wget' for completeness here?

Reply:
Good question. Also I don't show installation of docker or curl. I'll add those steps

Reply:
IMO, for common stuff, it is OK to simply have a list of pre-reqs, not take responsibility for documenting how to install them.